spec/javascripts/Policies/components/CustomPolicy.spec.jsx

```diff
@@ -22,7 +20,19 @@ describe('CustomPolicy', () => {
     expect(wrapper.find('header').hasClass('CustomPolicy-header')).toBe(true)
     expect(wrapper.find('h2').hasClass('CustomPolicy-title')).toBe(true)
     expect(wrapper.find('a').hasClass('CustomPolicy-cancel')).toBe(true)
-    expect(wrapper.find('Form').prop('initialPolicy')).toBe(POLICY_TEMPLATE)
+    expect(wrapper.find('CustomPolicyEditor').prop('initialPolicy')).toBe(POLICY_TEMPLATE)
+  })
+})
+
+describe('CustomPolicyEditor', () => {
+  it('should render itself correctly', () => {
+    const wrapper = shallow(<CustomPolicyEditor initialPolicy={POLICY_TEMPLATE} />)
+    expect(wrapper.find('SchemaEditor').exists()).toBe(true)
+    expect(wrapper.find('Form').exists()).toBe(true)
+    expect(wrapper.find('.PolicyConfiguration-name').text()).toBe('Name of the policy')
```

Comment:
Instead of testing against a string, you could test against `POLICY_TEMPLATE.schema.name`

Reply:
I could, but tests should as explicit as they can. That way if the `POLICY_TEMPLATE` changes somehow and we are not expecting that change, the test will fail.